ci/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM nvidia/cuda:12.0.0-devel-ubuntu20.04
+
+# julia
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive \
+    apt-get install --yes --no-install-recommends \
+                    # basic stuff
+                    curl ca-certificates \
+                    libcutensor-dev \
+                    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG JULIA=1.6
```

Comment:
Does this mean Julia v1.6. Is that what we want?

Reply:
Still working on getting this ironed out, I think this is only the default value and this is actually replaced in the Jenkinsfile. This is mostly just copied from the ITensors.jl repository and doesn't currently work yet.